roles/foreman_repositories/tasks/redhat.yml

```
@@ -48,8 +48,8 @@
       register: disabled_repos
       changed_when: false
 
-    - name: Enable powertools
+    - name: Enable PowerTools
```

```suggestion
    - name: Enable powertools
```

just to limit the diff noise :)